### ISSUE TITLE
[Question] encrypted data

### ISSUE THREAD

**Issue by wtercato** [firefly-iii/firefly-iii#1073]
I am running Firefly III version 4.6.11.1

Hello. How can I configure the system so that the database records and attached files in the transactions are not encrypted?

**Comment by JC5** [comment]
There is a variable in your `.env`-file: `USE_ENCRYPTION=true`. You can set this to `false`. Then run the following command on the command line:

```
php artisan firefly:use-encryption
```

Not all fields will be decrypted: some will remain encrypted. Attachments and file uploads will never be stored decrypted. The fields that will be unencrypted are:

* Account names
* Bill names and the words they match
* Budget names
* Category names
* Piggy bank names
* Transaction journal descriptions

Various other fields are either encrypted or unencrypted: this has to to with the history of Firefly III.

**Comment by wtercato** [comment]
Hello.

For a new installation, started from zero, if I turn off the encryption, nothing will be encrypted? except the files?

**Comment by JC5** [comment]
Yes. Some fields will always be encrypted. 

* Table `accounts`, field IBAN is always encrypted.
* Table `attachments`, fields filename, title, description and notes are always encrypted.
* Table `preferences`, field data is always encrypted
* Table `tags`, field tag is always encrypted

**Comment by jinformatique** [comment]
Hello James,
I recently upgraded to 4.7.12 and all the transactions got displayed encrypted (see pic).
![image](https://user-images.githubusercontent.com/487038/52913694-f2324080-32c0-11e9-8f49-ebdf5c7cfc6e.png)

I ran the decryption command and now it's good. All appears fine.
Now I would like to encrypt all the database again. What command should I run?
I tried:
` php artisan firefly:encrypt-file`
Not enough arguments (missing: "file, key").

In the documentation, this point was not mentioned.
In the .env file, if I turn `USE_ENCRYPTION=true` is that enough?
